decidim-proposals/app/forms/decidim/proposals/proposal_report_form.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+module Decidim
+  module Proposals
+    # A form object to be used when public users want to report a proposal.
+    class ProposalReportForm < Decidim::Form
+      mimic :proposal_report
+
+      attribute :reason, String
```

Comment:
I was following the design but I don't have a strong opinion on this. Thoughts @oriolgual ?